devel/addon/certmanager/install.sh

```diff
@@ -53,21 +54,22 @@ kubectl get namespace "${NAMESPACE}" || kubectl create namespace "${NAMESPACE}"
 # Build the Helm chart package .tgz
 bazel build //deploy/charts/cert-manager
 
-# Pre-compile the kubectl plugin, so it can quickly check the api status
-bazel build //hack/bin:kubectl-cert_manager
-
 # Upgrade or install cert-manager
+# --wait & --wait-for-jobs flags should wait for resources and Jobs to complete
 helm upgrade \
     --install \
     --wait \
+    --wait-for-jobs \
     --namespace "${NAMESPACE}" \
     --set image.tag="${APP_VERSION}" \
     --set cainjector.image.tag="${APP_VERSION}" \
     --set webhook.image.tag="${APP_VERSION}" \
+    --set startupapicheck.image.tag="${APP_VERSION}" \
```

Comment:
You probably should make these same changes in the `hack/verify-upgrade.sh` script.

```
$ make verify_upgrade
...

$ kubectl get pods -A
NAMESPACE            NAME                                         READY   STATUS         RESTARTS   AGE
cert-manager         cert-manager-5b4b8574f8-gwvkk                1/1     Running        0          42s
cert-manager         cert-manager-cainjector-6c5c7f6c47-2czng     1/1     Running        0          42s
cert-manager         cert-manager-startupapicheck-kvght           0/1     ErrImagePull   0          42s
cert-manager         cert-manager-webhook-76c4f7d5f8-7p5mk        1/1     Running        0          42s

```